gremlin-server/src/test/java/org/apache/tinkerpop/gremlin/driver/remote/GraphBinaryRemoteGraphComputerProvider.java

```diff
@@ -143,10 +143,14 @@
         test = "org.apache.tinkerpop.gremlin.process.traversal.step.map.PathTest",
         method = "g_injectX1_null_nullX_path_dedup",
         reason = "The inject() step is not supported by GraphComputer")
+@Graph.OptOut(
```

Comment:
do all the tests need to opted out? only the `GreedyMatchTraversals` use that strategy syntax, no?

Reply:
right, OptOut reduced to `MatchTest$GreedyMatchTraversals`